lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -457,9 +459,10 @@ defmodule RealtimeWeb.RealtimeChannel do
   end
 
   @impl true
-  def terminate(reason, _state) do
+  def terminate(reason, %{transport_pid: transport_pid}) do
     Logger.debug("Channel terminated with reason: #{reason}")
     :telemetry.execute([:prom_ex, :plugin, :realtime, :disconnected], %{})
+    Tracker.untrack(transport_pid)
```

Comment:
I don't think we have a guarantee that this will run if the RealtimeChannel process crashes 🤔

Reply:
the only way would be for us to actively monitor every channel in the tracker process which will be bad 😅 

or am I thinking wrong?